### PR TITLE
[MI-1745] hide shopify url in parameters

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations

### Description
* Hidden and remove required flag for shopify url. If the user manually sets the value to a non valid shopify url, the setup button will not appear. There is also no need for the url to be entered by the user as it can always be overridden after they are redirected to the setup url.
* Removed the deprecated `manifest.json` on the root folder

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-1745

### Risks
* [low] Changing the type to hidden also means removing the url validation

### CODE REVIEW

**Comment by jmramos02** [comment]
update risks please

File: dist/manifest.json [hunk 1]
```diff
@@ -19,8 +19,7 @@
   "parameters": [
     {
       "name": "url",
-      "type": "url",
-      "required": true
```
Comment (jmramos02) [inline dist/manifest.json]: Why are we removing this?

Comment (jmramos02) [inline dist/manifest.json]: ah got it, nvm

Comment (kcasas) [inline dist/manifest.json]: read the description ^

**Pull request merged**